course-details.md

```diff
@@ -1,13 +1,28 @@
 This course will dive into how you can get your best work done by identifying when and how to request a review, how to perform a review for someone else's pull request, and other awesome collaboration methods.
 
-All great projects start with collaboration. Pull requests are the foundation of teamwork on GitHub—and pull request reviews give you the ability to work together and discuss changes specific to a pull request by commenting, requesting changes, or approving.
+All great projects start with collaboration. Pull requests are the foundation of teamwork on GitHub — and pull request reviews give you the ability to work together and discuss changes specific to a pull request by commenting, requesting changes, or approving.
 
-In this course, you’ll learn how to:
+## What you'll learn
 
+We'll answer common questions like:
+- What is a pull request?
+- How do you comment on a pull request?
+- How do you review a pull request?
+- How do you request a pull request review?
+- How do you accept changes introduced by a pull request review?
```

Comment:
I've been using this section as an opportunity to add details to things that can't be covered in a learning outcome (which is the section below), which are very specifically about the activities the learner completes. Scanning through the responses might help. For this course, for example, I'd maybe add some of the following questions:
- What are the responsibilities of a reviewer?
- What's a `CODEOWNERS` file?
- What's a `diff` and how do I use it? 
- How do I implement changes from a pull request review? 
- How can I suggest a change when reviewing a pull request?

Reply:
done